imag-todo/src/ui.rs

```diff
@@ -0,0 +1 @@
+//ui
```

Comment:
Perfect, thats pretty much what i wanted when i said to open a Pull Request as soon as you start working.
However, we already got a merge conflict, because the file ui.rs already exists in the current branch.
To resolve this, do the following:
rm ui.rs // If you havent done work yet - ill just assume this right now
git checkout imag-task
git pull --ff-only <my_repo> imag-task // updates your imag-task branch
git checkout <this-branch>
git rebase imag-task
git push --force origin imag-task
Also, please do remember the next time you create a new branch, to update to the current master (in this case "imag-task", as it is our feature branch):
git checkout imag-task
git pull --ff-only <my_repo> imag-task
git checkout -b <new working branch>